### DISCUSSION TITLE
Screen saver with Immich photos on IOS

### DISCUSSION THREAD

**Discussion by owpac** [immich-app/immich#11257] [General]
Hello!

It would be awesome to have a screen saver with Immich photos on IOS!

I know there is some kind of alternative for computers with #8065, but it can't be used for phones (IOS ~nor Android~).

Thanks for all the good work with Immich!

**Comment by bo0tzz** [comment]
I believe ImmichFrame has an Android build. @3rob3 got any plans for iOS? :P

**Reply by owpac** [reply]
You're right about the Android build, my bad!

**Reply by 3rob3** [reply]
> I believe ImmichFrame has an Android build. @3rob3 got any plans for iOS? :P

It would be easy enough to add iOS, but a few things have kept me from pursuing:
- The ridiculous Apple developer fee. 
- No sideloading, so no easy way around the above.
- iOS doesn't support screensavers (I had a tvOS build working when I started the project, but abandoned it when I realized I would never be able to set it as a screensaver).

**Reply by owpac** [reply]
I see. I understand the two first point. For the third, maybe there is another approach than screensavers: widgets and lock screen with photos? 🤔

WDYT?

**Reply by bo0tzz** [reply]
> The ridiculous Apple developer fee.

If you set up github sponsors I'll chip in a little bit (and most likely others too)
